detox/jest.startup.js

```diff
@@ -15,22 +15,18 @@ beforeAll(async () => {
             detoxPrintBusyIdleResources: "YES",
             detoxURLBlacklistRegex: ".*firestore.*"
         },
-        permissions: { faceid: "YES", location: "inuse", camera: "YES", photos: "YES", notifications: "YES" }
+        permissions: { faceid: "YES", location: "inuse", camera: "YES", photos: "YES" }
     });
-    if (device.getPlatform() === "ios") {
-        await device.setBiometricEnrollment(true);
-    }
     await waitFor(element(by.id("NativeHome.Widgets")))
         .toBeVisible()
-        .withTimeout(120000);
-}, 120000);
+        .withTimeout(300000);
+}, 300000);
```

Comment:
Was this increased timeout intended to stay?

Reply:
Yes, Travis was failing drastically with dev bundle